google-cloud-spanner/lib/google/cloud/spanner/commit.rb

```diff
@@ -330,6 +286,40 @@ def mutations
 
         protected
 
+        ##
```

Comment:
Probably best to mark this one `@private` to ensure it doesn't show up in any documentation, since it's internal.